pom.xml

```diff
@@ -9,9 +10,15 @@
   </parent>
 
   <groupId>com.hotels</groupId>
-  <artifactId>housekeeping</artifactId>
-  <version>1.0.4-SNAPSHOT</version>
+  <artifactId>housekeeping-parent</artifactId>
+  <version>2.0.0-SNAPSHOT</version>
   <inceptionYear>2016</inceptionYear>
+  <packaging>pom</packaging>
+
+  <modules>
+    <module>housekeeping</module>
+    <module>housekeeping-vacuum-tool</module>
```

Comment:
OK, so you did split this into only two modules, I think the vacuum-tool pom then just has the wrong name inside it, it should be hk-vacumm-tool not hk-vacuum-tool-parent.